RELEASE_NOTES.md

```diff
@@ -1,3 +1,39 @@
+## Cookstyle 5.20
+
+### 1 New Cop
+
+#### ChefDeprecations/DeprecatedChefSpecPlatform
+
+The `ChefDeprecations/DeprecatedChefSpecPlatform` cop detects the usage of platforms in ChefSpec tests that have been deprecated in the Fauxhai / ChefSpec projects. Using these older platforms will cause you specs to error as mock Ohai data cannot be loaded to emulate a full Chef Infra Client run.
```

Comment:
```suggestion
The `ChefDeprecations/DeprecatedChefSpecPlatform` cop detects platforms in ChefSpec tests that have been deprecated in the Fauxhai / ChefSpec projects. Specifying deprecated platforms will create errors in ChefSpec because the mock Ohai data cannot be loaded to emulate a full Chef Infra Client run.
```